app/code/Magento/InventoryImportExport/etc/export.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0"?>
+<!--
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+-->
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:module:Magento_ImportExport:etc/export.xsd">
+    <entity name="stock_sources" label="Stock Sources" model="Magento\InventoryImportExport\Model\Export\Sources" entityAttributeFilterType="stock_sources"/>
+</config>
```

Comment:
additional blank line should be added at the end

Reply:
done